Cargo.toml

```diff
@@ -15,23 +15,21 @@ readme = "README.md"
 
 iron = "^0.5.0"
 #handlebars = { path = "../handlebars", default-features = false }
-handlebars = { version = "^0.25.0", default-features = false }
-rustc-serialize = { version = "^0.3.16", optional = true }
-serde = { version = "^0.9.0", optional = true }
-serde_json = { version = "^0.9.0", optional = true }
+handlebars = { version = "^0.26.1", default-features = false }
+serde = { version = "^1.0.0" }
+serde_json = { version = "^1.0.0" }
 plugin = "^0.2.6"
 notify = { version = "^3.0", optional = true }
 log = "^0.3"
 walkdir = "^1.0"
-serde_derive = { version = "^0.9.0", optional = true }
+serde_derive = { version = "^1.0.0", optional = true }
 
 
 [features]
-default = ["handlebars/default", "rustc-serialize"]
+default = ["handlebars/default"]
 watch = ["notify"]
-serde_type = ["handlebars/serde_type", "serde", "serde_json"]
 partial_legacy = ["handlebars/partial_legacy"]
-unstable = ["serde_type", "serde_derive"]
+unstable = ["serde_derive"]
```

Comment:
unstable feature is for travis build, which I think is no longer needed since proc_macro landed.
Also consider move `serde_derive` to dev-dependencies.